### PR TITLE
fix: wp-gatsby should checkout tags

### PR DESCRIPTION
**Pull request by acao**
this will fix today's issues with snapshots in CI

### CODE REVIEW

**Comment by TylerBarnes** [comment]
Looks like there may be some other changes needed. Tests are failing with `Cloning into '/usr/src/wordpress/wp-content/plugins/wp-gatsby'...
fatal: not a git repository (or any parent up to mount point /var/www)`

File: docker/wordpress/Dockerfile [hunk 1]
```diff
@@ -32,7 +32,7 @@ COPY install-plugin.sh /usr/local/bin/install-plugin
 RUN install-plugin wp-graphql wp-graphql/wp-graphql ${WPGRAPHQL_VERSION}
 
 # Install wp-gatsby using git, and apply a diff
-RUN git clone https://github.com/gatsbyjs/wp-gatsby ${PLUGIN_DIR}/wp-gatsby
+RUN git clone https://github.com/gatsbyjs/wp-gatsby ${PLUGIN_DIR}/wp-gatsby && git checkout ${WPGATSBY_VERSION}
```
Comment (TylerBarnes) [inline docker/wordpress/Dockerfile:35]: I think we need to cd into the directory before checking out the tag

**Review by TylerBarnes — CHANGES_REQUESTED** [review]

File: docker/wordpress/Dockerfile (hunk 1 above)
Comment (TylerBarnes) [reply docker/wordpress/Dockerfile]: Or we can clone the tag directly https://stackoverflow.com/a/21699307/12777907

Comment (TylerBarnes) [reply docker/wordpress/Dockerfile]: --depth=1 is probably a good idea anyway :)

**Comment by acao** [comment]
@TylerBarnes yep! already committed the fix :D

File: docker/wordpress/Dockerfile (hunk 1 above)
Comment (acao) [reply docker/wordpress/Dockerfile]: i think i already resolved this, i just pushed the commit as you were reviewing! oops haha

Comment (TylerBarnes) [reply docker/wordpress/Dockerfile]: Perfect :D

**Comment by TylerBarnes** [comment]
Woo! Thanks for sorting this out so quickly

**Pull request merged**